src/styles/globals.css

```
@@ -145,10 +145,11 @@
 }
 
 @utility step {
-  @apply max-md:ml-11 max-md:-indent-11;
+  @apply max-md:pl-10;
   counter-increment: step;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)